src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -112,8 +93,81 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     selectPortfolioCryptoHumanBalanceByFilter(s, feeAssetBalanceFilter),
   )
 
+  const selectedCurrency = useAppSelector(selectSelectedCurrency)
+
+  const getEstimateFeesArgs: () => Promise<EstimateFeesInput> = useCallback(async () => {
+    const amountCryptoBaseUnit = bnOrZero(state?.deposit.cryptoAmount).times(
+      bn(10).pow(asset.precision),
+    )
+    const quote = await getThorchainSaversQuote(asset, amountCryptoBaseUnit)
+
+    return {
+      cryptoAmount: state?.deposit.cryptoAmount ?? '',
```

Comment:
guard against 0 value amounts? or do it elsewhere?

Reply:
2d26b0bca34cdd8f7888fb33897afe4add84b91b